FluentMigrationRunner.cs

```diff
@@ -24,7 +24,9 @@ private static IServiceProvider CreateServices()
                     // Set the connection string
                     .WithGlobalConnectionString("Server=127.0.0.1;Port=3306;Database=simulation;Uid=simulationUser;Pwd=JabbatheSimulator1078;")
                     // Define the assembly containing the migrations
-                    .ScanIn(typeof(AddLogTable).Assembly).For.Migrations())
+                    .ScanIn(typeof(AddLogTable).Assembly).For.Migrations()
+                    .ScanIn(typeof(AddPlayerTable).Assembly).For.Migrations()
+                    .ScanIn(typeof(Migration2020042001).Assembly).For.Migrations())                    
```

Comment:
it looks like this ScanIn method takes a array; so you could reduce the verbosity via `.ScanIn(typeof(AddLogTable).Assembly,typeof(AddLogTable).Assembly...`
You might still want to put each table/migration on its own line to keep the easy scanning/readability, but it might make it a little less clunky.